packages/contracts-bedrock/package.json

```diff
@@ -29,7 +29,7 @@
     "validate-spacers": "pnpm build && pnpm validate-spacers:no-build",
     "slither": "./scripts/slither.sh",
     "slither:triage": "TRIAGE_MODE=1 ./scripts/slither.sh",
-    "clean": "rm -rf ./artifacts ./forge-artifacts ./cache ./tsconfig.tsbuildinfo ./tsconfig.build.tsbuildinfo ./scripts/go-ffi/go-ffi",
+    "clean": "rm -rf ./artifacts ./forge-artifacts ./cache ./tsconfig.tsbuildinfo ./tsconfig.build.tsbuildinfo ./scripts/go-ffi",
```

Comment:
this is somewhat unrelated to CI, but it was in the course of debugging this PR that I discovered a bug in this path.